### ISSUE TITLE
Makedatasets: option to pad images < target size, and downsize images > target size

### ISSUE THREAD

**Issue by dbuscombe-usgs** [dbuscombe-usgs/segmentation_zoo#26]
when presented with a given TARGET_SIZE, makedatasets could optionally be used to pad (with zeros) any image/label pair that are smaller than TARGET_SIZE

Images larger than TARGET_SIZE would still be downsized, like now

**Comment by dbuscombe-usgs** [comment]
not supported by `tf.keras.preprocessing.image.ImageDataGenerator` 

one option would be to modify the images in place ... before being passed to `tf.keras.preprocessing.image.ImageDataGenerator`

**Comment by dbuscombe-usgs** [comment]
I think this should be a stand-alone utility that creates a new set of images that are padded

**Comment by dbuscombe-usgs** [comment]
I have made a new utility function (`utils/pad_folder_of_images.py`) that will read each image and label pair, and create new copies of each that are either padded, or downsized by preserving the aspect ratio, then padded. The result is an image with TARGET_SIZE that preserves the aspect ratio of every image, regardless of size

**Comment by dbuscombe-usgs** [comment]
this is now provided as utils/pad_folder_of_images.py
